contracts/contract_caller.go

```diff
@@ -0,0 +1,67 @@
+package contracts
```

Comment:
this file is usually generated, what was the reason to edit it?

this is missing:
```
// Code generated - DO NOT EDIT.
// This file is a generated binding and any manual changes will be lost.
```

and please use more specifc name for contract. multiple contracts will conflict with each other if they will be stored in same directory. please use `contracts/registry`